students/seunghun/users/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+
+class User(models.Model):
+    name = models.CharField(max_length=10)
+    email = models.CharField(max_length=45)
+    password = models.CharField(max_length=20)
+    phone_number = models.CharField(max_length=20)
+    address = models.CharField(max_length=50, null=True)
+    date_of_birth = models.DateField(null=True)
+    nickname = models.CharField(max_length=10,null=True)
```

Comment:
```suggestion
class User(models.Model):
    name         = models.CharField(max_length=45)
    email        = models.CharField(max_length=200)
    password     = models.CharField(max_length=200)
    phone_number = models.CharField(max_length=45)
    address = models.CharField(max_length=200, null=True)
    birth_date = models.DateField(null=True)
    nickname = models.CharField(max_length=45, null=True)
```
1. max_length는 넉넉하게 지정해주는 것이 좋습니다.
2. 변수명은 명확하고 간결하게 작성해주세요.
3. 위와 같이 모든 속성을 제일 긴 변수명 기준으로 정렬해주세요.

Reply:
넵 수정완료했습니다 ㅎㅎ